src/js/features/player-keyboard-shortcuts.js

```diff
@@ -0,0 +1,21 @@
+var keyCodes = require('../keycodes');
+
+module.exports = function() {
+    function handleKeyEvent(keyup) {
+        if (!$('input, textarea, select').is(':focus')) {
+            if (keyup.keyCode === keyCodes.k) {
+                document.getElementsByClassName('js-control-playpause-button')[0].click();
```

Comment:
small nit, I'd prefer you to grab the player by id before traversing its children for the classes of the buttons.. Twitch uses `player` for the player's id..

`$('#player').find('.js-control-playpause-button').click()`

etc..